### PR TITLE
results/discussion revision

### PR DESCRIPTION
**Pull request by davharris**
I revised the results to match the new values (mainly changed due to the removal of the geometric distribution). I also changed up the order to discuss likelihood before AICc instead of after, because it's simpler and forms the basis for AICc.

### CODE REVIEW

**Comment by davharris** [comment]
I've now tightened up the discussion section. It's entirely possible that I discarded something important, though.

**Comment by ethanwhite** [comment]
@rueuntal or @embaldridge, why don't you give this a quick look. If you like what you see go ahead and merge and then @rueuntal can take the next round.

**Comment by rueuntal** [comment]
OK, I'll work on it this morning.

File: chapter1.md [hunk 1]
```diff
@@ -67,37 +67,31 @@ Model fitting, log-likelihood, and AICc calculations were performed using the ma
...
-
-The similar absolute fits of these five commonly used distributions emphasizes the challenges of inferring the processes operating in ecological systems from the form of the abundance distribution. It is already well established that models based on different processes can yield equivalent models of the SAD, i.e., they predict distributions of exactly the same form [@cohen1968, @pielou1975, @boswell1971, @mcgill2007]. It is also possible for the same biological explanations to result in  different forms of the species abundance distribution depending on community conditions [@hughes1986]. Our results support the idea that even when models do differ in their precise mathematical predictions that they are often not distinguishable enough to identify potential mechanisms with any degree of certainty [@volkov2005]. In other words, it is difficult to distinguish among the different distributions used to characterize the SAD, let alone the processes that generate the form of a particular distribution.
```
Comment (rueuntal) [inline chapter1.md]: I'm wondering if we want to discard or keep these paragraphs on SAD vs mechanism. Dave, what are your thoughts?

**Comment by rueuntal** [comment]
It's merged. I'll take the next round.

**Pull request merged**